src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -58,66 +56,89 @@ const ParentFamilyHome = props => {
   };
 
   return (
-    <>
-      <ParentSidebar />
-      <Content
-        className="family-page-container"
-        style={{
-          backgroundImage: `url(${cloudbg})`,
-        }} //background image here while troubleshooting LESS rendering issue
+    <Content
+      className="family-page-container"
+      style={{
+        backgroundImage: `url(${cloudbg})`,
+      }} //background image here while troubleshooting LESS rendering issue
```

Comment:
But if you do keep the styling here, do you need double { for the body of 'style' ?